client-src/elements/chromedash-report-external-reviews-page.js

```diff
@@ -0,0 +1,247 @@
+                      <sl-skeleton effect="sheen"></sl-skeleton>
+                    </td>
+                    <td class="milestones">
+                      <sl-skeleton effect="sheen"></sl-skeleton>
+                    </td>
+                  </tr>
+                `
+              )}
+            </table>
+          </section>
+        `,
+        complete: ({reviews, links, noOutstandingReviews}) =>
+          noOutstandingReviews
+            ? html`No outstanding reviews. Congratulations!`
+            : [
+                ['Preparing to ship', 'shipping'],
+                ['In Origin Trial', 'origin-trial'],
+                ['Getting wide review', 'wide-review'],
+                ['In developer trials', 'dev-trial'],
+                ['Prototyping', 'prototyping'],
+                ['Incubating', 'incubating'],
+                ['Already shipped', 'shipped'],
+              ].map(([title, key]) =>
+                reviews[key].length > 0
+                  ? html`<section>
+                      <h2 id=${key}>${title}</h2>
+                      <table class="data-table">
+                        ${this.headerRow()}
+                        ${reviews[key].map(
+                          /** @param {OutstandingReview} review */ review => html`
+                            <tr>
+                              <td class="feature">
+                                <a href="/feature/${review.feature.id}"
+                                  >${review.feature.name}</a
+                                >
+                              </td>
+                              <td class="review">
+                                <chromedash-link
+                                  href=${review.review_link}
+                                  .featureLinks=${links}
+                                ></chromedash-link>
+                              </td>
+                              <td class="milestones">
+                                ${review.estimated_start_milestone
+                                  ? 'M' + review.estimated_start_milestone
+                                  : nothing}${['shipping', 'shipped'].includes(
+                                  review.current_stage
+                                )
+                                  ? nothing
+                                  : html`${review.estimated_start_milestone ||
+                                    review.estimated_end_milestone
+                                      ? '–'
+                                      : nothing}${review.estimated_end_milestone
+                                      ? 'M' + review.estimated_end_milestone
+                                      : nothing}`}
+                              </td>
+                            </tr>
+                          `
+                        )}
+                      </table>
+                    </section>`
+                  : nothing
+              ),
+        error: e => {
+          console.error(e);
```

Comment:
Do you want to keep this console.error?

Reply:
Yes, since this branch results in a useless page, and people who encounter it should have a message they can paste to us in their bug report. But I should give it a better error message.